src/UI/Text.elm

```diff
@@ -100,6 +102,16 @@ colorPrimary =
         ( Palette.tonePrimary, Palette.lumMiddle )
 
 
+colorBgMiddle : TextColor
+colorBgMiddle =
+    Internal.ColorBgMiddle
+
+
+colorInherit : TextColor
+colorInherit =
+    Internal.ColorInherit
```

Comment:
You're giving the developer god-like powers :scream_cat: 
![iddqd](https://thumbs.gfycat.com/HighlevelWealthyHarrier-size_restricted.gif)